testing-with-jest.test.js

```diff
@@ -7,67 +7,21 @@
 
 
 // --------------------1a) Create a test for a function called addThemUp that takes two numbers as arguments and returns the sum.
-//*********
-// Don't laugh but I forgot the "quotes" on the describe portion of my test and made the test work by having the initialization of function addThemUp so my test knew what to call on... I see how useful strings are here in finding throughout the entire document something called "addThemUp"
-// var numbers1 = 24
-// var numbers2 = 22
-// var numbers3 = numbers1 + numbers2
-// const addThemUp = (numbers1, numbers2) =>{
-//   return numbers3
-// }
-//*********
-// this test below FAILS
-
-// describe("addThemUp", () =>{
-//   test("Takes two numbers as arguments and returns the sum", () =>{
-//     expect(addThemUp()).toEqual()
-//   })
-// })
-// now just add in the two numbers as arguments to equal the third var
-describe("addThemUp", () =>{
-  test("Takes two numbers as arguments and returns the sum", () =>{
-    expect(addThemUp(24, 22)).toEqual(46)
-  })
-})
 
 
-// --------------------1b) See the test fail. THEN write the code to make the test pass.
-//this is where I honestly feel like its cheating? I define the summation of the two numbers as a variable and call upon it... I definitely need way more practice!
```

Comment:
Nope, you're doing it mostly right. You dont need to declare the variables:

```javascript
var numbers1 = 24
var numbers2 = 22
```

As this is redundant. The test passes those numbers in for you, the function just has to exist.

Reply:
Now I understand!